code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -283,12 +277,8 @@ Auto Patrol: []"},
 				return
 
 			if(iscarbon(target) && target.canBeHandcuffed())
-				if(!arrest_type)
-					if(!target.handcuffed)  //he's not cuffed? Try to cuff him!
-						cuff(target)
-					else
-						back_to_idle()
-						return
+				if(!target.handcuffed)  //he's not cuffed? Try to cuff him!

```

Comment:
Please keep the detain function, remove it from the interface & topic calls, allow mappers/admins to use it if they wish.